components/brave_wallet/common/brave_wallet.mojom

```diff
@@ -1102,6 +1159,50 @@ interface AssetRatioService {
   GetCoinMarkets(string vs_asset, uint8 limit) => (bool success, array<CoinMarket> values);
 };
 
+interface BuyAndSellService {
```

Comment:
Is there a better namer for that new service? None of its methods has eiter buy or sell in their names